templates/zerver/for/open-source.md

```diff
@@ -0,0 +1,99 @@
+
+## Great for developers
+
+* Messages take Markdown formatting, [including code
+  blocks](/help/format-your-message-using-markdown#code).  Super easy
+  to discuss code, paste error messages, etc.
+
+* Want to refer to issues or code reviews with short notation like
+  `#1234`, `T1234` -- or any other style of your own?  Set up any
+  regex as a [custom linkification
+  filter](/help/add-a-custom-linkification-filter) for your
+  organization.
+
+* Have a design discussion in chat and want to link to it in your bug
+  tracker or a commit message?  [Two
+  clicks](/help/share-a-message-or-conversation) makes a permanent
+  link.  The URL transparently shows where the discussion happened,
+  and for everyone who clicks through, Zulip's topic threading
+  provides a focused view on that conversation.
+
+* Troubleshooting in chat for users or new developers becomes much
+  more manageable with Zulip's topic threading: each issue in a
+  channel can be discussed in its own thread.
+
+## Great for productivity
+
+* With Zulip's unique model of threading by topic, your community can
+  have more than one conversation in a channel at the same time,
+  without the confusion and talking-over caused by traditional models
+  like IRC and Slack.  (Extra handy for holding a synchronous chat
+  meeting with 5-20 people and making good use of everyone's time!)
+
+* No limited "try it out" tier -- [host it
+  yourself](https://zulip.readthedocs.io/en/latest/prod-install.html),
+  or [sign up at zulipchat.com](https://zulipchat.com/beta/) for
+  commercially-supported hosting, free for open source projects.
+
+* Fast, full-text search through your full, unlimited message history.
+
+* [Stop by
+  chat.zulip.org](https://zulip.readthedocs.io/en/latest/chat-zulip-org.html),
+  Zulip's own development community, to see it in action for a large
+  project with ~5000 messages/week, essentially all of it productive
+  developer discussion.
+
+## Great for open communities
+
+* Sleek, modern [apps for every platform](/apps/) make Zulip an
+  inclusive choice for everyone in your community.
+
+* No need to invite people one by one into your community, or use a
+  third-party bot to do so -- [one
+  checkbox](/help/allow-anyone-to-join-without-an-invitation) opens
+  your Zulip organization for anyone to join.
+
+* Want to link to your Zulip organization with a nice badge in your
+  README?  [Done.](/help/join-zulip-chat-badge)
+
+<!--- This needs better documentation. -->
+* GitHub auth?  [We've got that.][github-auth]
+
+[github-auth]: https://github.com/zulip/zulip/blob/7e9926233/zproject/prod_settings_template.py#L112
+
+* Zulip scales well to large communities -- it's loved by communities
+  as large as 1000 users and beyond.
+
+## Zulip ❤​ open source
+
+* Zulip itself is open source, [under the Apache License 2.0](https://github.com/zulip/zulip/blob/master/LICENSE).
+
+* Full, commercially-supported hosting at zulipchat.com ([signup
```

Comment:
"You can sign up your community **today** for free, full, commercially-supported..."